client/extensions/zoninator/components/data/query-zones/index.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isFetchingZones } from '../../../state/zones/selectors';
+import { fetchZones } from '../../../state/zones/actions';
```

Comment:
We had some naming discussions on this sort of (network fetching) action in Calypso, and I think we settled on `request...` over `fetch...` at some point 😉 (Unless I'm mistaken and there's yet another different convention for `data-layer` middleware based requests, that is.)